### DISCUSSION TITLE
How to create an alias that goes to /dev/null ?

### DISCUSSION THREAD

**Discussion by himred** [postfixadmin/postfixadmin#821] [Q&A]
Hi !

I tried but I'm only allowed to enter an email address as alias destination.
Any idea how I could do that ?

Thanks for your help

**Comment by DavidGoodwin** [comment]
I think postfixadmin always wants an email address as a destination; if you want something else, as a one off, it'd probably be easiest to change postfix's main.cf to include a custom lookup - for example

```
...
virtual_alias_maps = hash:/etc/postfix/custom pgsql:/etc/postfix/mysql/virtual_alias_maps.cf
...
```

and custom could then contain e.g.

```
somename: /dev/null
some@name: /dev/null
```

etc?
